doc/releasenotes/15_0_2_changelog.md

```diff
@@ -0,0 +1,27 @@
+# Changelog of Vitess v15.0.2
+
+### Bug fixes 
+#### Query Serving
+ * Online DDL: fix 'vtctlclient OnlineDDL' template queries [#11889](https://github.com/vitessio/vitess/pull/11889)
+ * Fix CheckMySQL by setting the correct wanted state [#11895](https://github.com/vitessio/vitess/pull/11895)
+ * bugfix: allow predicates without dependencies with derived tables to be handled correctly [#11911](https://github.com/vitessio/vitess/pull/11911)
+ * [release-15.0] Fix sending a ServerLost error when reading a packet fails (#11920) [#11930](https://github.com/vitessio/vitess/pull/11930)
+ * Skip `TestSubqueriesExists` during upgrade-downgrade tests [#11953](https://github.com/vitessio/vitess/pull/11953) 
```

Comment:
Should we really list CI config changes in the release notes?

Reply:
Optimally no, but as of now we don't have a system to filter PRs in the changelog.

Reply:
However, we should manually go through the summary during the release and clean up irrelevant rows